Homepage Update September 2021

kylekirkby opened the pull request:
Homepage Update September 2021

EbbaSimpson reviewed (approved):
Looks great, thanks Kyle! The only thing I saw was an M that needs removing:

**M**Value of Linaro contributed code to GCC, LLVM and QEMU in 2020

And I'm assuming the order of the logos has changed so the Qualcomm logo is bigger?

Thanks,

Ebba

kylekirkby commented:
> Looks great, thanks Kyle! The only thing I saw was an M that needs removing:
> 
> **M**Value of Linaro contributed code to GCC, LLVM and QEMU in 2020
> 
> And I'm assuming the order of the logos has changed so the Qualcomm logo is bigger?
> 
> Thanks,
> 
> Ebba

Hi @EbbaSimpson ,

I've changed the order that members are displayed in and removed that pesky "M"  :)
